ssh.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+echo "Generating a new SSH key for GitHub..."
+
+# Generating a new SSH key
+# https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#generating-a-new-ssh-key
+ssh-keygen -t ed25519 -C "dries@vints.io"
```

Comment:
Instead of adding my email address to the source code, can we use a parameter I need to pass here instead? So I explicitly need to pass my email address using the bash command in the instructions.

Reply:
You're thinking about a flag or just the `$1` argument?

Reply:
Thinking an argument indeed.